R/render-plot.R

```diff
@@ -37,6 +37,7 @@ renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
   } else {
     installExprFunction(expr, "func", env, quoted)
   }
+  func <- wrapFunctionLabel(func, "renderPlot", ..stacktraceon = TRUE)
```

Comment:
Yeah it sure might. Do we want third party devs to be doing this too? It's getting complicated...

Maybe we should add this behavior as an additional argument to installExprFunction?